assets/js/actions/alert.js

```diff
@@ -0,0 +1,21 @@
+import * as rest from '../util/rest';
+
+export const createAlert = (alert) => {
+  return (dispatch) => {
+
+    return rest.post('/api/alerts', {
+        alert: alert
+      })
+      .then(response => {})
+  }
+}
+
+export const deleteAlert = (id) => {
+  return (dispatch) => {
+
+    return rest.post('/api/alerts/delete', {
```

Comment:
we can make this a delete action right? instead of a post?

Reply:
router.ex will need to be changed too

Reply:
you right you right